--- reframe/core/buildsystems.py	2024-01-02 10:04:50.862512667 +0000
+++ reframe/core/buildsystems.py	2024-01-02 10:03:56.249990969 +0000
@@ -876,10 +876,10 @@
     def emit_build_commands(self, environ):
         ret = self._create_env_cmds()
 
-        if self._auto_env:
-            install_tree = self.install_tree or 'opt/spack'
-            ret.append(f'spack -e {self.environment} config add '
-                       f'"config:install_tree:root:{install_tree}"')
+        # if self._auto_env:
+        #     install_tree = self.install_tree or 'opt/spack'
+        #     ret.append(f'spack -e {self.environment} config add '
+        #                f'"config:install_tree:root:{install_tree}"')
 
         if self.specs:
             specs_str = ' '.join(self.specs)
